index.html

```diff
@@ -22,7 +22,54 @@
         <!--"header end"-->
     
         <!--"main-block start"-->
-        <section id="main-block"></section>
+        <section id="main-block">
+            <div class="background-container" id="main-block">
+        <div class="main">
+            <div class="h1">
+                <p class="title-h1">ДОБРОГО ВЕЧОРА, МИ З УКРАЇНИ!</p>
+            </div>
+            <div class="title">
+                <p class="predact">
+                    Програма працевлаштування ветеранів, ВПО та громадян, які постраждали
+                    внаслідок війни росії проти України та українського народу.
+                </p>
+            </div>
+            <div class="suffered" style="margin-top: 30px;">
+                <button type="button" class="Signup">Зареєструватися</button>
+            </div>
+        </div>
+    </div>
+
+    <div id="main-block">
+        <div class="pointsL">
+            <div class="Vector Vector1">
+                <img src="../b23_dv_dark2/src/Photo/Vector.png" alt="Vector Image" />
+            </div>
+            <div class="Vector Vector2">
+                <img src="../b23_dv_dark2/src/Photo/Vector.png" alt="Vector Image" />
+            </div>
+        </div>
+
+        <div class="wemade">
+            <p class="wemade-title">
+                Ми створили і постійно розвиваємо всесвітню мережу офлайн/онлайн
+                центрів підтримки українців, що постраждали внаслідок війни,
+                розв’язаної росією проти України та її народу. А також центрів
+                підтримки ветеранів та громадян, які змушені були покинути місця свого
+                постійного проживання.
+            </p>
+        </div>
+
+        <div class="pointsR">
```

Comment:
Possibly pick a different class name? What is pointsR in this context?

Reply:
" - points in right side **pointsR** and 
" - points in left side **pointsL**